resources/views/index.blade.php

```diff
@@ -67,8 +66,8 @@
       <h2 class="c-top-heading-h2"><span>ABOUT US</span>Playlistについて</h2>
       <div class="p-top-about_pic">
         <picture>
-          <source srcset="{{asset('img/top/aboutus-pc.png')}}" media="(min-width:640px)" type="image/png">
-          <img class="u-top-img" src="{{asset('img/top/aboutus.png')}}" alt="">
+          <source srcset="{{asset('img/top/aboutus-pc.jpg')}}" media="(min-width:640px)" type="image/jpg">
```

Comment:
@KomatsuHikaru 
```suggestion
          <source srcset="{{asset('img/top/aboutus-pc.jpg')}}" media="(min-width:640px)" type="image/jpeg">
```
拡張子がjpgでも、typeは`jpeg`（eが入っている）じゃないとダメらしいです…
ややこしい仕様😭

Reply:
対応しました。画像の丸角対応も同時にコミットしています。[コミット先](https://github.com/KeisukeHoshino/PlaylistLP/commit/1d2a2330130b85aa0d636d44bd988100240eba46)